config/user_data.go

```diff
@@ -2,31 +2,19 @@ package config
 
 import (
 	"fmt"
-
-	"gopkg.in/yaml.v3"
+	"time"
 
 	"github.com/buildwithgrove/path/config/utils"
 )
 
+const defaultCacheRefreshInterval = 5 * time.Minute
+
 /* --------------------------------- User Data Config Struct -------------------------------- */
 
 // UserDataConfig contains user data configuration settings.
```

Comment:
#PUC this comment. 

What's the goal of this structure from a functional perspective?

Reply:
Just contains the DB connection string and an optional refresh interval (with default of 5 mins). Updated comment.